elf_reader.go

```diff
@@ -647,6 +659,15 @@ func (ec *elfCode) relocateInstruction(ins *asm.Instruction, rel elf.Symbol) err
 
 			ins.Constant = 0
 
+		case ksymsVar != nil && ins.OpCode.IsDWordLoad():
+			if bind != elf.STB_GLOBAL {
+				return fmt.Errorf("asm relocation: %s: %w: %s", name, errUnsupportedBinding, bind)
+			}
+			println("==> set")
+			ins.Metadata.Set(ksymMetaKey{}, &ksymMeta{
+				rel.Name,
+			})
```

Comment:
`__ksym`s can also be `__weak` meaning that we should not error when we can't find them. Can we imediatly add support for this? Please look at the `case`s above this one to see how we handled it for kfuncs.

Reply:
I've added support for `__weak` as well 👍